packages/core/src/components/datetime/datetime.tsx

```diff
@@ -219,7 +219,10 @@ export class AtomDatetime {
                 datetime={this.datetimeId}
                 disabled={this.disabled}
                 mode='md'
-              ></ion-datetime-button>
+              >
+                <slot name='date-target' />
+                <slot name='time-target' />
```

Comment:
question: why do we need this? I didn't quite understand the use case

Reply:
It's related to this [ion component](https://github.com/juntossomosmais/atomium/pull/754). It allows the customization of the value shown in the button.